charts/loki-stack/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: "v1"
 name: loki-stack
-version: 2.6.4
+version: 2.6.5
```

Comment:
This one is 2.6.6

Reply:
fixed in b4099bcaa1d21729f9e8e3c15ba801feb3088d6f